dpnp/dparray.pyx

```diff
@@ -942,6 +942,34 @@ cdef class dparray:
         """
         return argsort(self, axis, kind, order)
 
+    def partition(arr, kth, axis=-1, kind='introselect', order=None):
```

Comment:
use `self` for accessing the attributes and methods of the instance of the class.